specs/networking/rpc-interface.md

```diff
@@ -225,6 +225,7 @@ Requests a list of block roots and slots from the peer. The `count` parameter MU
     start_slot: uint64
     max_headers: uint64
     skip_slots: uint64
+    backward: uint8 in {0, 1}
```

Comment:
Should this be of type `bool`?

Reply:
Yes, semantically this is a `bool`. Spec-wise though I didn't find any occurrences of `bool` to date, and went kinda the same way it was done in [eth1 wire spec](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#getblockheaders-0x03). Should I change it to bool instead?

Reply:
We use `bool` in https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md

Reply:
Indeed. Changed to `bool`